classes/forms/student_submission_form.php

```diff
@@ -57,7 +57,7 @@ public function definition() {
         $this->add_hidden_elements_to_form();
 
         // if TII plagiarism enabled check if user agreed/disagreed EULA
-        if (!$this->get_coursework()->plagiarism_enbled() || has_user_seen_tii_EULA_agreement()) {
+        if (!$this->get_coursework()->plagiarism_enbled() || has_user_seen_tii_eula_agreement()) {
```

Comment:
good we're changing it noe but a bit concerning it didn't do it consistently across the board when this was renamed in a previous MR

Reply:
There is a rule for changing function names but it didn't work that well so I adjusted this manually